### ISSUE TITLE
Icey Discord Theme

### ISSUE THREAD

**Issue by Wintrier** [birdlinux/Icey-Discord-Theme#1]
The minimalistic member list is not functioning as intended.

**Comment by birdlinux** [comment]
Yeah, i know. I havent updated this in a While. You can check my dotfiles here

https://github.com/0romos/CosOS-dotfiles/

for updated stuff
